api/pom.xml

```diff
@@ -145,6 +145,11 @@
          <groupId>org.apache.lucene</groupId>
          <artifactId>lucene-queries</artifactId>
       </dependency>
+      <dependency>
+				<groupId>mysql</groupId>
+				<artifactId>mysql-connector-java</artifactId>
+				<scope>test</scope>
+			</dependency>
```

Comment:
@reagan-meant Couple of things ,

1. Please format the code before committing. Please use [this ](https://wiki.openmrs.org/display/docs/Developer+How-To+Setup+And+Use+Eclipse#DeveloperHow-ToSetupAndUseEclipse-eclipse-java-code-formatJavaCodeFormat) formatter.

2. Also i think this dependency is not necessary. I've cloned your branch and removed this dependency and built it. It worked fine. Can you please try `mvn clean install ` and see whether you are getting the same error ?

Reply:
Yeh, totally agree.